02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2036 @@
+{
```

Comment:
<p>Instead of "we can manipulate" it might be worth swapping to something like "we will need to manipulate" or "we should manipulate"</p><p>The current sentence suggests the below cell will somehow supply us with a note or tag that says whether or not it can be manipulated, whereas really, users would do this to check to see if they have any work to do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Great connotative point, I changed this to the following:</p><p> </p><p><strong>Great, looks like we're set up. Let's now take a look at the list of our hierarchies to see which ones we will look to manipulate.</strong></p>